image/image.go

```diff
@@ -86,7 +86,7 @@ func (li *localInfos) Info(name string) *snap.Info {
 	return nil
 }
 
-func localSnaps(tsto *ToolingStore, opts *Options) (*localInfos, error) {
+func localSnaps(tsto *ToolingStore, opts *Options, model *asserts.Model) (*localInfos, error) {
```

Comment:
do we need the model here now?

Reply:
Thanks, we don't! I removed it now again.